app/features-json/build_json_controller.rb

```diff
@@ -59,8 +59,6 @@ class BuildJSONController < APIController
         return
       end
 
-      branch_to_trigger = "master" # TODO: how/where do we get the default branch
-
       git_fork_config = GitForkConfig.new(
         sha: current_sha,
         branch: branch_to_trigger,
```

Comment:
It looks like we're not defining `branch_to_trigger` anywhere else, so this `GitForkConfig` would be initialized with a `nil` `branch:` parameter.